src/main.js

```diff
@@ -744,13 +748,42 @@ class MiniGraphCard extends LitElement {
 
   updateBounds({ config } = this) {
     this.bound = [
-      this.getBoudary('min', config.lower_bound, this.primaryYaxisSeries, this.bound[0]),
-      this.getBoudary('max', config.upper_bound, this.primaryYaxisSeries, this.bound[1]),
+      this.getBoundary('min', config.lower_bound, this.primaryYaxisSeries, this.bound[0]),
+      this.getBoundary('max', config.upper_bound, this.primaryYaxisSeries, this.bound[1]),
     ];
+
+    if (config.min_bound_range) {
+      const minBoundRange = parseFloat(config.min_bound_range);
+      const currentRange = Math.abs(this.bound[0] - this.bound[1]);
+      const diff = minBoundRange - currentRange;
+
+      // Doesn't matter if minBoundRange is NaN because this will be false if so
+      if (diff > 0) {
+        this.bound = [
+          this.bound[0] - diff / 2,
+          this.bound[1] + diff / 2,
+        ];
+      }
```

Comment:
Should make this into a function, since it's basically the same code repeated over again for the secondary axis.
Would also be useful if we decide to add support for more/unlimited axes in the future.

Reply:
I felt like the increase in complexity to factor this out was much higher than the increased utility. If you support unlimited values it'd make sense to do this then. There would likely be a lot of changes associated with that change.